discovery-provider/src/queries/queries.py

```diff
@@ -196,7 +194,19 @@ def get_tracks_including_unlisted():
         current_user_id = get_current_user_id(required=False)
         extended_tracks = populate_track_metadata(session, track_ids, tracks, current_user_id)
 
-    return api_helpers.success_response(extended_tracks)
+        filtered_extended_tracks = []
+
+        def check_unlisted_track_for_proper_route_id(track):
+            if track["is_unlisted"] and track["route_id"] != helpers.create_track_route_id(i["url_title"], i["handle"]):
```

Comment:
Hmm, don't think this is going to work since we don't have access to `i` at this point! I'm guessing you tested this locally, but the code didn't break because it was comparing `route_id` to an undefined response from `create_track_route_id` or something?

Reply:
oh shoot you're right 😣i'll make the proper change